packages/components/src/internal/components/forms/input/CheckboxInput.tsx

```diff
@@ -87,29 +88,34 @@ class CheckboxInputImpl extends DisableableInput<CheckboxInputProps, CheckboxInp
     };
 
     render() {
-        const { allowDisable, label, name, queryColumn, showLabel, addLabelAsterisk } = this.props;
+        const { allowDisable, label, name, queryColumn, showLabel, addLabelAsterisk, renderFieldLabel } = this.props;
         const { isDisabled } = this.state;
 
         // N.B.  We do not use the Checkbox component from Formsy because it does not support
         // React.Nodes as labels.  Using a label that is anything but a string when using Checkbox
         // produces a "Converting circular structure to JSON" error.
         return (
             <div className="form-group row">
-                <FieldLabel
-                    label={label}
-                    labelOverlayProps={{
-                        isFormsy: false,
-                        inputId: queryColumn.name,
-                        addLabelAsterisk,
-                    }}
-                    showLabel={showLabel}
-                    showToggle={allowDisable}
-                    column={queryColumn}
-                    isDisabled={isDisabled}
-                    toggleProps={{
-                        onClick: this.toggleDisabled,
-                    }}
-                />
+                {renderFieldLabel
+                    ? <label className={'control-label col-sm-3 text-left col-xs-12'}>
```

Comment:
```suggestion
                    ? <label className="control-label col-sm-3 text-left col-xs-12">
```
Using double quotes when possible for className makes it possible for IntelliJ to find the classes in the scss files.

Reply:
Ah yes, thanks. Done